src/getting_started/installing_rust.md

```diff
@@ -3,7 +3,19 @@
 Nannou is a library written for the [Rust programming
 language](https://www.rust-lang.org/). Thus, the first step is to install Rust!
 
-To install Rust, open up your terminal, copy the text below, paste it into your
+Installing Rust depends on the following packages:
+* `bash`
+* `build-essentials`
+* `python`
+* `cmake`
+* `wget` or `curl`
```

Comment:
Thanks for the PR!

I believe the `python` and `cmake` packages are only dependencies for shaderc (the thing that compiles GLSL shaders into SPIR-V so that they can be run with the Vulkan API) and are not required to install Rust, though I could be wrong!

I'm not 100% sure about `build-essentials`, but I believe it is only relevant to Debian/Ubuntu and can probably be omitted from this section.

I don't think we should have to mention `curl` as it comes with stock macos and windows, and it's kind of implied that we need it as the following command is a `curl` command to download Rust.

I feel similarly about `bash` - it comes with stock macos and almost every linux distro and I'm not sure it's actually required on windows?

I guess in general, I don't feel that this section is necessary in this chapter. Perhaps instead, after the command below, we can refer users to the [Rust Getting Started](https://www.rust-lang.org/learn/get-started) section if they run into any issues with the Rust installation process? Also, it *would* be worth adding the `python` and `cmake` dependencies to the "platform-specific dependencies" chapter under Vulkan if they're not already there, but I think they might already be mentioned. @freesig what are your thoughts?

Reply:
FreeBSD for example gives you neither `bash` nor `curl` by default. Yes, at least `curl` is implied, but I would prefer having all dependencies mentioned upfront before I have to fail through multiple steps to collect them.

Reply:
My only concern is unnecessarily complicating this chapter with platform-specific build requirements when we already have a platform-specific chapter. Perhaps we could instead move the platform-specific chapter *before* the Rust installation chapter so that it may include the platform-specific requirements for building Rust as well?

Reply:
I moved them.